kargo/kargo-htb.js

```diff
@@ -165,13 +166,16 @@ function KargoHtb(configs) {
 
     function __getUserIds(returnParcels) {
         var crb = __getCrb();
+        var privacyEnabled = ComplianceService.isPrivacyEnabled();
+        var uspConsentObj = ComplianceService.usp && ComplianceService.usp.getConsent();
 
         return {
             kargoID: crb.userId || '',
             clientID: crb.clientId || '',
             tdID: __getTDID(returnParcels),
             crbIDs: crb.syncIds || {},
-            optOut: crb.optOut || false
+            optOut: crb.optOut || false,
+            usp: privacyEnabled && uspConsentObj ? uspConsentObj.uspString : null
```

Comment:
@samuelhorwitz

When the IX Library is not able to retrieve CCPA information the `ComplianceService.usp.getConsent()` function will return an object with the `uspString` property set to an empty string (ex. `{version: 1, uspString: ""}`).

In that case, can you confirm what the expected value for `usp` will need to be (null or empty string)?

If `usp` value is expected to be `null` then the value of `uspConsentObj.uspString ` will need to be checked before assigning it to the `usp` property.

Reply:
Our backend will handle null and empty string equivalently for the IAB string

Reply:
Perfect. In that case the changes look good and we will move forward with merging into master.